float64[3] point
---
int8 ok